revgen/app.go

```diff
@@ -116,21 +115,11 @@ VERSION: {{.Version}}
 func findRootDir(configFileName string) *string {
 	currDir, err := os.Getwd()
 	check(err)
-	out, err := runCmd("git rev-parse --show-toplevel", &currDir)
+	_, err = runCmd("git rev-parse --show-toplevel", &currDir)
 	check(err)
-	rootDir := strings.TrimSpace(out)
-
-	for {
-		filename := filepath.Join(currDir, configFileName)
-		if _, err := os.Stat(filename); errors.Is(err, os.ErrNotExist) {
-			currDir = filepath.Dir(currDir)
-		} else {
-			return &currDir
-		}
-
-		if currDir == rootDir || currDir == "." {
```

Comment:
The bug is somewhere here, I would expect currDir == rootDir to hit and exit the for loop, but it doesn't.